base/src/main/java/com/blazebit/expression/base/LocalDateOperatorInterpreter.java

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright 2019 - 2021 Blazebit.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blazebit.expression.base;
+
+import com.blazebit.domain.runtime.model.DomainOperator;
+import com.blazebit.domain.runtime.model.DomainType;
+import com.blazebit.domain.runtime.model.TemporalInterval;
+import com.blazebit.expression.ComparisonOperator;
+import com.blazebit.expression.DomainModelException;
+import com.blazebit.expression.ExpressionInterpreter;
+import com.blazebit.expression.spi.ComparisonOperatorInterpreter;
+import com.blazebit.expression.spi.DomainOperatorInterpreter;
+
+import java.io.Serializable;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneId;
+
+public class LocalDateOperatorInterpreter
```

Comment:
Can you please name the classes `Date*` instead of `LocalDate*` to align with the rest of the naming?